tpl/bootstrap.php

```diff
@@ -578,7 +578,7 @@
 					<h4 class="col-md-5 col-xs-8"><?php echo I18n::_($NAME); ?> <small>- <?php echo I18n::_('Because ignorance is bliss'); ?></small></h4>
 					<p class="col-md-1 col-xs-4 text-center"><?php echo $VERSION; ?></p>
 					<p id="aboutbox" class="col-md-6 col-xs-12">
-						<?php echo I18n::_('%s is a minimalist, open source online pastebin where the server has zero knowledge of pasted data. Data is encrypted/decrypted <i>in the browser</i> using 256 bits AES. More information on the <a href="https://privatebin.info/">project page</a>.', I18n::_($NAME)), PHP_EOL; ?>
+						<?php echo I18n::_('%s is a minimalist, open source online <a href="https://privatebin.info">pastebin</a> where the server has zero knowledge of pasted data. Data is encrypted/decrypted <i>in the browser</i> using 256 bits AES.', I18n::_($NAME)), ' ', I18n::_($INFO), PHP_EOL; ?>
```

Comment:
You would need to add a `%s` where the info needs to be inserted. And we need to split the translations in all of the translation files into two key/value sets.

I would not add a second link to the project url as I imagine that whoever replaces the info may not want a link to the url.

Reply:
Fixed. Thanks!